Google.PowerShell.IntegrationTests/Logging/Logging.GcLogSink.Tests.ps1

```diff
@@ -194,14 +199,14 @@ Describe "New-GcLogSink" {
         }
     }
 
-    It "should work with -NoUniqueWriterIdentity" {
+    It "should work with -UniqueWriterIdentity" {
```

Comment:
Has this cmdlet been released already? Are we concerned about backwards compatibility?